CHANGELOG.md

```diff
@@ -1,5 +1,79 @@
 # Release Notes
 
+## Version 1.21.0-beta.0 - 2021-06-02
+Features:
+* Support setting addons from environmental variables [#11469](https://github.com/kubernetes/minikube/pull/11469)
+* Add "resume" as an alias for "unpause" [#11431](https://github.com/kubernetes/minikube/pull/11431)
+* Implement target node option for cp command [#11304](https://github.com/kubernetes/minikube/pull/11304)
```

Comment:
add back ticks for `cp`
or maybe word it like this

`cp ` command: add target node

Reply:
done